src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,169 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertSame;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static StorageDao storageDao;
+    private static User userFirst;
+    private static User userSecond;
+    private static final String MODEL_FIRST_USER_LOGIN = "Andrew";
+    private static final String MODEL_FIRST_USER_PASSWORD = "Andrew123";
+    private static final int MODEL_FIRST_USER_AGE = 19;
+    private static final String MODEL_SECOND_USER_LOGIN = "Bohdan";
+    private static final String MODEL_SECOND_USER_PASSWORD = "Bohdan123";
+    private static final int MODEL_SECOND_USER_AGE = 23;
```

Comment:
What 'MODEL' means in all these constants?

Reply:
Model = Model case for tests